module/VuFind/src/VuFind/ILS/Driver/KohaILSDI.php

```diff
@@ -210,6 +210,32 @@ protected function initDb()
     }
 
     /**
+     * Check if a table exists in the current database.
+     *
+     * @param string $table Table to search for.
+     *
+     * @return bool
+     */
+    function tableExists($table)
+    {
```

Comment:
Would it make sense to add a static cache here, so if an operation is repeated many times it doesn't cause a large number of extra queries?